test/testSyntax.js

```diff
@@ -1,4 +1,4 @@
-require('source-map-support').install()
+'use strict'
```

Comment:
'use strict' isn't necessary in sources transpiled with babel; since we're using offside/jsy syntax in this test, we'll be using babel.